packages/sdk/src/Utils.ts

```diff
@@ -110,7 +110,7 @@ const convertToMapById = (entityMap: CLM.FilledEntityMap): CLM.FilledEntityMap =
 export const actionHasHash = (actionId: string, hash: string, actions: CLM.ActionBase[]) => {
     // Check that correct action was taken
     const action = actions.find(a => a.actionId === actionId)
-    if (action?.clientData?.actionHashes) {
+    if (action ?.clientData ?.actionHashes) {
```

Comment:
Hmm, I see some of the spaces removed on other files but still see spaces here. I"m guessing it only fixed the files you opened and saved since you fixed the formatter. Also should update others to remove spaces.

Reply:
I think github comments can be stale sometimes, did you fix this?

Reply:
What's the status here, I thought you were wanting to get this merged?  I don't see this comment as outdated and I still see bad formatting.  If someone else opens this file later these will show as changes they made because of autoformat even though they didn't change them.

Reply:
fixed